wezterm/src/main.rs

```diff
@@ -228,7 +232,12 @@ fn run() -> anyhow::Result<()> {
 
     let opts = Opt::from_args();
     if !opts.skip_config {
-        config::reload();
+        if let Some(ref config_file) = opts.config_file {
```

Comment:
I would have tackled this a bit differently:

* Added a `static ref CONFIG_FILE_OVERRIDE: Mutex<Option<PathBuf>> = Mutex::new(None)` `config/src/lib.rs`, and then add a function to set it:

```rust
pub fn set_config_file_override(path: &Path) {
   CONFIG_FILE_OVERRIDE.lock().unwrap().replace(path.to_path_buf());
}
```

Then here in main.rs the logic would be:

```rust
if let Some(config_file) = opts.config_file.as_ref() {
   config::set_config_file_override(config_file);
}
config::reload();
```

then see my comment in `config/src/lib.rs` for where I'd hook that up

Reply:
done!